Gemfile.lock

```diff
@@ -79,6 +79,11 @@ GEM
   specs:
     CFPropertyList (3.0.0)
     abstract_type (0.0.7)
+    activesupport (5.2.0)
```

Comment:
As pointed out by @snatchev, this is not perfect, but not much we can do for now. I'll investigate on why this is added, and submit an issue if there isn't one already